framework/src/transfers/MultiAppVariableValueSamplePostprocessorTransfer.C

```diff
@@ -223,6 +223,8 @@ MultiAppVariableValueSamplePostprocessorTransfer::execute()
         mooseError("MultiAppVariableValueSamplePostprocessorTransfer does not support transfer of "
                    "vector variables");
 
+      _fe_problem.setActiveElementalMooseVariables({&_var}, 0);
```

Comment:
```suggestion
      _fe_problem.setActiveElementalMooseVariables({&_var}, /*thread_id=*/0);
```
same for the other

how do we know this variable is elemental?

Reply:
Not familiar with other types of variables. Would adding `if (_var->isElemental())` work?

Reply:
dont think we have that one. what are you trying to ban exactly here? variables missing ghosting?

Reply:
Primarily yes, although this is not a perfect solution. How about `if (!_var->isNodal())`

Reply:
will have to do. If we want a hasDofsOutsideNodes here